tests/rptest/tests/random_node_operations_test.py

```diff
@@ -530,9 +530,6 @@ def cluster_version_updated():
                 err_msg="Error waiting for cluster to report consistent version"
             )
 
-        if with_tiered_storage:
-            self.redpanda.stop_and_scrub_object_storage()
-
         # Validate that the controller log written during the test is readable by offline log viewer
         log_viewer = OfflineLogViewer(self.redpanda)
```

Comment:
As a side effect of this PR controller log validation will finally work. The stop_and_scrub call from above would result in the `self.redpanda.start_nodes()` call below to return an empty list...

Wondering if this is why offline log viewer wasn't working and nobody noticed until https://github.com/redpanda-data/redpanda/pull/23803

Reply:
nice